### PR TITLE
#1109: Improve Pit Coverage

### PR DESCRIPTION
**Pull request by goldbal330**
Signed-off-by: Alexander Goldberg <alexander.goldberg@bnymellon.com>

### CODE REVIEW

File: unit-tests/src/test/java/org/eclipse/collections/impl/block/CheckedBlocksTest.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021 Goldman Sachs.
+ * Copyright (c) 2021 Goldman Sachs and others.
```
Comment (donraab) [inline unit-tests/src/test/java/org/eclipse/collections/impl/block/CheckedBlocksTest.java:2]: Change year to 2022

**Review by donraab — CHANGES_REQUESTED** [review]
Hi @goldbal330, I requested one small consistency change. Otherwise LGTM.

File: unit-tests/src/test/java/org/eclipse/collections/impl/block/CheckedBlocksTest.java (hunk 1 above)
Comment (goldbal330) [reply unit-tests/src/test/java/org/eclipse/collections/impl/block/CheckedBlocksTest.java]: Fixed. Thanks.

**Review by donraab — APPROVED** [review]
LGTM. Thanks for the contribution @goldbal330 !

**Pull request merged**